docs/docs/workflows/troubleshooting.md

```diff
@@ -55,14 +55,12 @@ When in doubt, check if the encountered [issue is known](https://github.com/edge
 
 :::
 
-
 :::tip
 
-During an upgrade with modified attestation config, a backup of the current configuration is stored in the `join-config-backup` config map in the `kube-system` namespace. To restore the old attestation config after a failed upgrade, you can copy the attestation config from this resource, put it in your configuration file and retry the upgrade.
+During an upgrade with modified attestation config, a backup of the current configuration is stored in the `join-config` config map in the `kube-system` namespace under the `attestationConfig_backup` key. To restore the old attestation config after a failed upgrade, replace the value of `attestationConfig` with the value from `attestationConfig_backup`. Alternatively you can copy the attestation config from `attestationConfig_backup`, put it in your configuration file and rerun `constellation upgrade apply`.
```

Comment:
Do you think there's any situation where the alternative action is preferable? If not, we should just scratch that sentence.

Reply:
@thomasten  I think letting the user change the config is safer because subsequent upgrade apply will try to apply the wrong config again.

Reply:
Ok so if changing the config is our preferred way, we should mention that first. (And then we probably wouldn't want to provide the kubectl command for the other approach.)
What do you think @daniel-weisse ?

Reply:
I agree that manual measurement management should not be a problem to the normal user. So I guess that either way is fine for troubleshooting